multicluster/controllers/multicluster/member_clusterset_controller.go

```diff
@@ -92,6 +92,9 @@ func (r *MemberClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		}
 		klog.InfoS("Received ClusterSet delete", "clusterset", req.NamespacedName)
 		if r.remoteCommonArea != nil {
+			if err := r.deleteMemberAnnounce(); err != nil {
```

Comment:
@luolanzone : do we have Finalizer on ClusterSet too? Here if mc-controller crashes after deletion failure, the memberAnnounce will be left in the leader for ever?